scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/signal/SignalApi.java

```diff
@@ -147,6 +147,7 @@
 
     /**
      * Remove the given {@code signal} from the set of job signals
+     * In order to remove a signal that was not yet received, the signalsSubSet parameter should contain "_ready" prefix and the signal name
```

Comment:
the "_ready" prefix

Reply:
ready suffix, no ?

Reply:
it's prefix, but it shoud be "ready_" not "_ready", I will change it. Thank you

Reply:
ah ok

Reply:
it's before the signal name: signalapi.removeSignal("ready_Stop_Job")